### ISSUE TITLE
Feature Request: Change service image listed on SONOS

### ISSUE THREAD

**Issue by japostoles** [simojenki/bonob#174]
It would be nice to be able to manage the image displayed next to the BNB_SONOS_SERVICE_NAME in the SONOS listing.

Is this possible, or does that image come from the upstream service (e.g. Navidrome)?  Currently in the S1 app I'm seeing a dotted square icon.

**Comment by simojenki** [comment]
Yes, this would be great, however unfortunately I do not think this is possible.  I think those images are assigned as part of the official on-boarding process that services go thru to be added to the Sonos app.

**Comment by dlehammer** [comment]
> Service logos are line art images that represent your service in the music catalog. Listeners see this logo when searching for your content service to add it to their Sonos system. They then see the logo once they've added your service and they want to use it.

Based on the above snippet from https://docs.sonos.com/docs/add-logos , it sounds possible 🤓

**Comment by simojenki** [comment]
Yes, this is where I got to and then decided it wasn't going to be straightforward
